selfdrive/car/gm/interface.py

```diff
@@ -219,11 +219,16 @@ def _get_params(ret, candidate, fingerprint, car_fw, experimental_long, docs):
       CarInterfaceBase.configure_torque_tune(candidate, ret.lateralTuning)
 
     elif candidate == CAR.SILVERADO:
-      ret.mass = 2200. + STD_CARGO_KG
+      ret.mass = 2500. + STD_CARGO_KG
       ret.wheelbase = 3.75
       ret.steerRatio = 16.3
       ret.centerToFront = ret.wheelbase * 0.5
       tire_stiffness_factor = 1.0
+      ret.steerActuatorDelay = 0.5
```

Comment:
@nuwandavek do you know if the two lateral changes are good changes? it really has this much delay and low lat accel?

Reply:
This seems to be not needed when using tim's NNFF, so this may not be the correct approach

Reply:
yeah, I'd split out the necessary changes (mass) from the delay. 0.5 seconds is a lot of delay - maybe there are other effects.